megago/make_GO_freq_json.py

```diff
@@ -1,29 +1,56 @@
 import json
+from goatools.anno.idtogos_reader import IdToGosReader
+from goatools.obo_parser import GODag
+from goatools.semantic import TermCounts
+import pkg_resources
+import os
+
+
+
+DATA_DIR = pkg_resources.resource_filename(__name__, 'resource_data')
+JSON_INDEXED_FILE_PATH = os.path.join(DATA_DIR, "go_freq_uniprot.json")

```

Comment:
This could be moved to the `__init__.py` file and loaded from there. Same for the `megago.py` file. That way, we only define this at a single place.
```suggestion
from . import JSON_INDEXED_FILE_PATH
```